core/src/main/scala/com/salesforce/op/stages/impl/feature/OPMapVectorizer.scala

```diff
@@ -397,7 +397,7 @@ final class DateMapVectorizerModel[T <: OPMap[Long]] private[op]
   val timeZone: DateTimeZone = DateTimeUtils.DefaultTimeZone
 
   def convertFn: DateMap#Value => RealMap#Value = (dt: DateMap#Value) =>
-    dt.mapValues(v => Days.daysBetween(new DateTime(v, timeZone), referenceDate).getDays.toDouble)
+    dt.mapValues(v => new Duration(new DateTime(v, timeZone), referenceDate).getStandardDays.toDouble)
```

Comment:
also, there are more places like this in the code. try grepping.

Reply:
done!

Reply:
there is at least one more in `DateTimeVectorizerTest`